In-house patch to remove the dependency on argparse.  Since we use
Python 2.7 only for OpenStack, argparse is already included.  If this
line is present, stevedore will fail to find the pip package for it,
causing a cascade of failures to other tools.

--- python-saharaclient-0.8.0/requirements.txt.orig	2015-08-02 02:14:21.878516359 -0400
+++ python-saharaclient-0.8.0/requirements.txt	2015-08-02 02:15:04.434964849 -0400
@@ -12,4 +12,3 @@
 requests>=2.2.0,!=2.4.0
 six>=1.7.0
 PrettyTable>=0.7,<0.8
-argparse

--- python-saharaclient-0.8.0/python_saharaclient.egg-info/requires.txt.orig	2015-08-02 17:17:40.108351859 -0400
+++ python-saharaclient-0.8.0/python_saharaclient.egg-info/requires.txt	2015-08-02 02:14:48.979346364 -0400
@@ -7,4 +7,3 @@
 requests>=2.2.0,!=2.4.0
 six>=1.7.0
 PrettyTable>=0.7,<0.8
-argparse
